bin/ch-builder2squash

```diff
@@ -36,18 +36,25 @@ if [ "$#" -lt 2 ]; then
 fi
 
 builder_choose
-echo "builder: ${CH_BUILDER}"
```

Comment:
What's the reasoning to remove this chatter?

Reply:
Because `ch-builder2squash` calls `ch-builder2tar` which already has that line so you would get a parrot of the builder selected.